netperf.spec.in

```diff
@@ -7,7 +7,7 @@ Group:         System Environment/Base
 License:       Unknown
 URL:           http://www.netperf.org/
 Packager:      Martin A. Brown
-Source:        ftp://ftp.netperf.org/netperf/%{name}-%{version}.tar.bz2
+Source:        https://github.com/HewlettPackard/%{name}/archive/master.tar.gz
```

Comment:
Matteo, are you sure building "master" is the right thing to do for all redhat based distros?
Or is the %{version} branch just missing from github?

Every distro I've ever used picked up the version (or "release branch"), not the development branch.

Reply:
Hrm...it looks like https://github.com/HewlettPackard/netperf/archive/netperf-2.7.0.tar.gz exists... so I'm curious why "master" is the preferred tarball here. TIA!

Reply:
Yes, usualy a tagged release should be used, but this is the master branch and refers to master, in other tags it should point to %{version}

Reply:
Ah! Yeah, that makes sense - thanks!